engine/src/globals.cpp

```diff
@@ -1266,6 +1266,9 @@ bool X_open(int argc, MCStringRef argv[], MCStringRef envp[])
 
 int X_close(void)
 {
+    // MW-2012-02-23: [[ FontRefs ]] Finalize the font module.
+    MCFontFinalize();
```

Comment:
I'm not quite sure of the dependencies here, but this should probably go after LogicalFontTableFinalize and before GraphicsFinalize. In theory Extensions could tear down loaded fonts in finalization.